File: CS/WindowsApplication1/CustomEditor/MyPopupBaseSizeableFormPainter.cs
using System;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.Registrator;
using DevExpress.XtraEditors.Popup;

namespace WindowsApplication1
{
    public class MyPopupBaseSizeableFormPainter : PopupBaseSizeableFormPainter
    {
        protected override void DrawContent(PopupFormGraphicsInfoArgs info)
        {
            base.DrawContent(info);
            MyCustomBlobPopupFormViewInfo vi = info.ViewInfo as MyCustomBlobPopupFormViewInfo;
            MyPopupContainerForm form = vi.Form as MyPopupContainerForm;
            Rectangle rect = form.myButton.Bounds;
            rect.X = rect.Right;
            rect.Width = vi.SizeGripRect.X - rect.X;
            info.Graphics.DrawString("MyPopupBaseSizeableFormPainter", info.ViewInfo.Appearance.GetFont(), Brushes.Red, rect);
        }

    }
}

File: VB/WindowsApplication1/CustomEditor/MyPopupBaseSizeableFormPainter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.Registrator
Imports DevExpress.XtraEditors.Popup

Namespace WindowsApplication1
	Public Class MyPopupBaseSizeableFormPainter
		Inherits PopupBaseSizeableFormPainter
		Protected Overrides Sub DrawContent(ByVal info As PopupFormGraphicsInfoArgs)
			MyBase.DrawContent(info)
			Dim vi As MyCustomBlobPopupFormViewInfo = TryCast(info.ViewInfo, MyCustomBlobPopupFormViewInfo)
			Dim form As MyPopupContainerForm = TryCast(vi.Form, MyPopupContainerForm)
			Dim rect As Rectangle = form.myButton.Bounds
			rect.X = rect.Right
			rect.Width = vi.SizeGripRect.X - rect.X
			info.Graphics.DrawString("MyPopupBaseSizeableFormPainter", info.ViewInfo.Appearance.GetFont(), Brushes.Red, rect)
		End Sub

	End Class
End Namespace
